src/components/card/Card.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../styles/variables';
+
+.bdl-Card {
+    background-color: $white;
+    border-radius: 3px;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
+    margin: 0 auto;
+    max-width: 900px;
+
+    .bdl-Card-title {
```

Comment:
Please avoid nesting classes unless necessary, as it increases specificity. The SUIT naming convention also makes it easier to avoid.